source/headers.ts

```diff
@@ -24,7 +23,7 @@ const getResetSeconds = (
 	resetTime?: Date,
 	windowMs?: number,
 ): number | undefined => {
-	let resetSeconds: number | undefined = undefined // eslint-disable-line no-undef-init
+	let resetSeconds: number | undefined
```

Comment:
We could probably make windowMs a required param, and then we wouldn't have to worry about undefined. The one place where we call `getResetSeconds()` without windowMs, we have windowMs available, so it should be a pretty easy change.

On the other hand, there's already a lot changing here, so maybe best to leave it for now...

Reply:
Yea, I'll fix this in the tests PR instead.

Reply:
Done in [6b33fe1](https://github.com/express-rate-limit/express-rate-limit/pull/509/commits/6b33fe183801f06907a1e1342f650da98c42858f).